department-of-veterans-affairs/abd-vro
# MCP-2128 Fix checkstyle errors in persistence and service modules

## What was the problem?
We have checkstyle warnings in the build logs for coding convention errors.

Associated tickets or Slack threads:

- [MCP-2128](https://amida.atlassian.net/browse/MCP-2128)

## How does this fix it?[^1]
Addresses the errors with the appropriate fixes and removes the errors from the logs.

## How to test this PR
- Step 1 clean and build abd-vro
- Step 2 verify there are no checkstyle warnings for service or persistence


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
